deploy/kubernetes/elastic-agent-kustomize/default/elastic-agent-standalone/base/elastic-agent-standalone-daemonset.yaml

```diff
@@ -28,19 +28,21 @@ spec:
       # Uncomment if using hints feature
```

Comment:
Maybe we could have another base for hints, instead of leaving it in a comment here.

Reply:
As we discuss lets tackle this in another story